Packs/ServiceNow/Integrations/ServiceNowEventCollector/ServiceNowEventCollector.py

```diff
@@ -377,9 +437,22 @@ def main() -> None:  # pragma: no cover
         if command == "test-module":
             return_results(module_of_testing(client, log_types))
 
-        elif command == "service-now-get-audit-logs" or command == "service-now-get-syslog-transactions":
-            log_type = AUDIT if command == "service-now-get-audit-logs" else SYSLOG_TRANSACTIONS
-            audit_logs, results = get_events_command(client=client, args=args, log_type=log_type, last_run=last_run)
+        elif command == "service-now-get-audit-logs":
```

Comment:
these 3 ifs are mostly duplicate code besides log_type, lets see if we can share some of the code

Reply:
done